packages/maskbook/src/plugins/ITO/hooks/usePoolPayload.ts

```diff
@@ -1,7 +1,7 @@
 import { PluginITO_RPC } from '../messages'
 import type { JSON_PayloadInMask } from '../types'
 
-const storage = new Map<string, JSON_PayloadInMask>()
+const storage = new Map<string, JSON_PayloadInMask | Error>()
 
 export function usePoolPayload(pid: string) {
     if (!storage.has(pid)) throw suspender(pid)
```

Comment:
We also should not throw error in usePoolPayload(), since we call usePoolPayload() outside <ITO /> and <ITO_LoadingFail />

Reply:
We can add more suspense

Reply:
Do not need more suspense.

```xml
<Suspense>
  <PostInspector> // usePoolPayload() 
    <ITO_LoadingFail>
      <ITO>
```